.env.example

```
@@ -1,24 +1,29 @@
+DECODO_PASS=
 
 MAIL_USERNAME=
 MAIL_PASSWORD=
 
-ASAAS_API_KEY=
-
-REDIS_PASSWORD=
 
-DECODO_USER=
-DECODO_PASS=
+OTEL_ENDPOINT_METRICS:http://otel-collector:4318/v1/metrics
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The OTEL_ENDPOINT_METRICS:http://otel-collector:4318/v1/metrics key has incorrect delimiter

(IncorrectDelimiter)